hack/tests/e2e-helm.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+#===================================================================
+# FUNCTION trap_add ()
+#
+# Purpose:  prepends a command to a trap
+#
+# - 1st arg:  code to add
+# - remaining args:  names of traps to modify
+#
+# Example:  trap_add 'echo "in trap DEBUG"' DEBUG
+#
+# See: http://stackoverflow.com/questions/3338030/multiple-bash-traps-for-the-same-signal
+#===================================================================
+trap_add() {
```

Comment:
I would prefer to make those consolidations in a follow on, I think it would be easier for reviews and we can then take the time to potentially design a nice API interface to the watches file that we can share. I think there is more to this then just sharing a template IMO. Thoughts?

Reply:
SGTM